plugin/src/main/java/org/opennms/velocloud/rest/api/VelocloudRestService.java

```diff
@@ -46,4 +51,24 @@ public interface VelocloudRestService {
     @Path("/partner/{alias}/customers")
     @Produces(value = {MediaType.APPLICATION_JSON})
     List<EnterpriseDTO> getCustomersForMspPartner(@PathParam("alias") String alias) throws VelocloudApiException;
+
+    @GET
+    @Path("/connections/list")
+    @Produces(value = {MediaType.APPLICATION_JSON})
+    List<ConnectionListElementDTO> getConnectionList() throws VelocloudApiException;
+
+    @POST
+    @Path("/connections/{alias}/add")
+    @Consumes({MediaType.APPLICATION_JSON})
+    Response addConnection(@PathParam("alias") String alias, @QueryParam("dryrun") boolean dryRun, ConnectionDTO connectionDTO) throws VelocloudApiException;
+
+    @POST
+    @Path("/connections/{alias}/edit")
```

Comment:
```suggestion
    @PUT
    @Path("/connections/{alias}")
```

`PUT` is the verb already.